youtube_dl/options.py

```diff
@@ -525,6 +525,11 @@ def _comma_separated_values_options_callback(option, opt_str, value, parser):
         '--external-downloader-args',
         dest='external_downloader_args', metavar='ARGS',
         help='Give these arguments to the external downloader')
+    downloader.add_option(
+        '--split-tracks',
+        dest='split_tracks', action='store_true', default=False,
+        help="Split tracks based on chapters information"
+    )
```

Comment:
I guess this option should be in `postproc` instead of `downloader`. isn't?

Reply:
Sounds reasonable, I'll have a look later